package.json

```diff
@@ -24,24 +36,10 @@
     "eslint-plugin-jsx-a11y": "^6.3.1",
     "eslint-plugin-react": "^7.21.2",
     "eslint-plugin-react-hooks": "^4.1.2",
+    "html-loader": "^1.3.1",
+    "html-webpack-plugin": "^4.5.0",
     "webpack": "^4.44.2",
-    "webpack-cli": "^3.3.12"
-  },
-  "scripts": {
```

Comment:
scripts are here in case you want to add them back in

Reply:
Thank you!